core/common/src/main/java/alluxio/worker/block/BlockStoreLocation.java

```diff
@@ -150,25 +152,17 @@ public boolean belongsTo(BlockStoreLocation location) {
    */
   @Override
   public String toString() {
-    StringBuilder result = new StringBuilder();
-    if (mDirIndex == ANY_DIR) {
-      result.append("any dir");
-    } else {
-      result.append("dir ").append(mDirIndex);
+    MoreObjects.ToStringHelper strHelper = MoreObjects.toStringHelper(this);
```

Comment:
Do we not want to show the `ANY` information anymore?

Reply:
It's implicit by not having it shown.
I couldn't find any justification to generate a human-readable string.
However this helps a great deal during investigations.

Reply:
Right, I would prefer explicit for easier debugging.

Reply:
Made it explicit, but with the new string helper format.